apps/core-app/src/main/modules/plugin/dev-server-monitor.test.ts

```
@@ -0,0 +1,113 @@
+}))
+
+vi.mock('../../utils/logger', () => ({
+  createLogger: () => ({
+    debug: vi.fn(),
+    info: vi.fn(),
+    warn: vi.fn(),
+    error: vi.fn()
+  })
+}))
+
+import { DevServerHealthMonitor } from './dev-server-monitor'
```

ESLint

[error] Import in body of module; reorder to top.

(import/first)